ddtrace/debugging/_capture/model.py

```diff
@@ -38,9 +38,9 @@ class CaptureState(object):
 
 @attr.s
 class CapturedEvent(six.with_metaclass(abc.ABCMeta)):
-    """Capture Event baseclass.
+    """Captured event base class.
 
-    Used to store collected when a probe is triggered.
+    Used to store captured data when a probe is triggered.
     """
 
     probe = attr.ib(type=Probe)  # type: Probe
```

Comment:
Should we define
```python
class ConditionalProbe(Probe, ProbeConditionMixin):
    pass
```
in the probe model and use that here? It seems that all the probes that we define there have the `ProbeConditionMixin`, so that can perhaps become the base class for all our conditional probes? Then we wouldn't have to do the casting here.

Perhaps we could also define an actual `LogProbe` as
```python
class LogProbe(ConditionalProbe, LogProbeMixin):
    pass
```
(maybe also rename `LogProbeMixin` into something else?) as I think this is what `CapturedEvent` actually deals with. WDYT?

Reply:
while I loved the idea. ConditionalProbe have the condition optional field - which require that mixing to always be the last in the chain.